editor/assets/stylesheets/_variables.scss

```diff
@@ -48,6 +48,10 @@ $admin-sidebar-width-big: 190px;
 $admin-sidebar-width-collapsed: 36px;
 $shadow-popover: 0px 3px 20px rgba( $dark-gray-900, .1 ), 0px 1px 3px rgba( $dark-gray-900, .1 );
 
+/* Blocks */
+$block-padding: 14px;
+$block-left-margin: 18px;
```

Comment:
Is this about the mover? Should we call it that considering it may move to the right in RTL?